pages/common/nix-env.md

```diff
@@ -2,6 +2,10 @@
 
 > Manipulate or query Nix user environments.
 
+- Show installed packages:
+
+`nix-env -q`
+
 - Show available package with name or without name:
```

Comment:
I don't understand this one. How can it show a package without a name ?

Reply:
I don't think that is part of this PR, but I think it should be `package(s)`. You can either list all packages, or ones matching the specified name query.

Reply:
Yeah, the description could be improved, but that's out of scope of this PR. That said, if @aaronduino wants to do it, it would be appreciated :)

Reply:
Yeah, it's not part of the PR. But now the first description becomes part of this, since that also shows installed packages. I think we need to change the description to remove the "without name" part. Also, I would like a clarification on the difference between `-q` and `-a`. If `-qa` is available and `-q` is installed, what is `-a` ?

Reply:
`-q` is shorthand for `--query`, `-a` is shorthand for `--available`.

Reply:
We should consider incorporating those terms in the descriptions (as per [guideline n.3](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines)); that would help with understanding and memorizing the command's syntax.

Reply:
@aaronduino - Could you update the PR with the changes ?

Reply:
How should I go about incorporating the word "query," considering that -q by itself does not let a user query the installed packages, but rather provides a full list of everything installed?

However, I'll definitely use the word "query" for the second and third examples.

Reply:
Should I submit changes to the second and third examples as a separate PR or just as separate commits?

Reply:
> How should I go about incorporating the word "query," considering that -q by itself does not let a user query the installed packages, but rather provides a full list of everything installed?

We can always use the [full version](https://nixos.org/nix/manual/#synopsis-6) `--query --instaled` (which is more verbose but not out of place in tldr-pages, since our mission is not to provide the shortest commands but understandable ones). Ideally we should be able to do `-qi` and use the spelled words in the description, but there isn't a short form of the `--installed` option :(

> Should I submit changes to the second and third examples as a separate PR or just as separate commits?

Additional commits in this PR are fine, thanks! 👍

Reply:
With the recent changes, I don't think using `--query --installed` is needed anymore, so I'll mark this conversation as resolved.